work/v1alpha1/types_manifestworkreplicaset.go

```diff
@@ -86,6 +90,23 @@ type LocalPlacementReference struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:MinLength=1
 	Name string `json:"name"`
+
+	// +optional
+	RolloutStrategy cluster.RolloutStrategy `json:"rolloutStrategy,omitempty"`
+}
+
+// PlacementSummary provides info regards number of clusters and clusterGroups selected by the placement refs.
+type PlacementSummary struct {
+	// PlacementRef Name
+	Name string `json:"name"`
+
+	// availableClusters shows number of clusters having manifestWorks available regards total number of clusters
+	// ex; 28/100 (28 out of 100)
+	AvailableClusters string `json:"availableClusters"`
+
+	// availableDecisionGroups shows number of decisionGroups having manifestWorks available regards total number of decisionGroups
+	// ex; 2/4 (2 out of 4)
+	AvailableDecisionGroups string `json:"availableDecisionGroups"`
```

Comment:
Can have a discussion here, whether need a common way to show the rollout progress. Today, addon shows this message in condition Progressing.
```
status:
  installProgression:
  - name: aws-placement
    namespace: default
    configReferences
...
    conditions:
    - lastTransitionTime: "2022-11-02T05:48:12Z"
      message: 200/400 upgrading...
      reason: Upgrading
      status: "True"
      type: Progressing
```

Reply:
Good to have common condition but I guess it hard to happen. Like in ManifestWorkReplicaSet there is no **progressing** condition neither **upgrading** reason.  As well as the Manifest has only apply/processing/available state.

Reply:
The [ManifestWorkReplicaSetSummary](https://github.com/open-cluster-management-io/api/blob/main/work/v1alpha1/types_manifestworkreplicaset.go#L93) showing the applied/progrssing/available/degraded state for all clusters manifestworks. The PlacementSummary is more to shows the placement rollout state. Not sure, Do we need to add manifestWorkSummary for each placementSummary  ?

Reply:
done, added the ManifestWorkReplicaSetSummary to the placementSummary so every placement will has its own summary info